airbyte-integrations/connectors/destination-snowflake/src/test-integration/kotlin/io/airbyte/integrations/destination/snowflake/SnowflakeTestDataComparator.kt

```diff
@@ -49,17 +49,23 @@ class SnowflakeTestDataComparator : AdvancedTestDataComparator() {
         }
     }
 
-    override fun compareDateTimeValues(expectedValue: String, actualValue: String): Boolean {
```

Comment:
sigh. we should really find a way to kill this class, it's basically an inferior RecordDiffer >.>

(iirc only used in DestinationAcceptanceTest)